Source/Magick.NET/Shared/Pixels/SafePixelCollection.cs

```diff
@@ -169,6 +170,12 @@ public override byte[] ToByteArray(MagickGeometry geometry, string mapping)
             return base.ToByteArray(geometry, mapping);
         }
 
+        public override byte[] ToByteArray(MagickGeometry geometry, PixelMapping mapping)
+        {
+            Throw.IfNull(nameof(geometry), geometry);
```

Comment:
I know it's a bit OCD but can you add an empty line between the throw and the return statement?